### DISCUSSION TITLE
FluentSelect dropdown Z-order -how to change?

### DISCUSSION THREAD

**Discussion by AndyEdmonds** [microsoft/fluentui-blazor#230] [Q&A]
I have a Teams app using the fluent blazor components. The dropdown area  of a fluentselect component falls behind other elements on the page. What css do I need to increase its z-order?

**Comment by vnbaaij** [comment]
I'll need to look into that. Will keep you posted (here).

**Reply by AndyEdmonds** [reply]
hi vnbaaij, any progress on this? 
Andy

**Reply by vnbaaij** [reply]
This turned out to be a deeper issue in the Fluent Web Components. The team is solving it there and once they are done, it will trickle down to the Blazor components as well. Don't have a timeline, I'm afraid.

**Reply by AndyEdmonds** [reply]
Thanks.
